File: listas/ex4.cpp
#include <stdio.h>

int main()
{
	int a[4],b[4],d[4][2],c,c2;
	printf("MATRIZ A:\n");
	for(c=0;c<4;++c)
	{
		printf("Digite o numero %i: ",c+1);
		scanf("%i",&a[c]);
	}
	printf("MATRIZ B:\n");
	for(c=0;c<4;++c)
	{
		printf("Digite o numero %i: ",c+1);
		scanf("%i",&b[c]);
	}
	for(c=0;c<2;++c)
	{
		for(c2=0;c2<4;++c2)
		{
			if(c==0)
			{
				d[c2][c] = a[c2]*2;
			}
			else
			{
				d[c2][c] = b[c2] - 5;
			}
		}
	}
	for(c=0;c<2;++c)
	{
		for(c2=0;c2<4;++c2)
		{
			printf(" [%i] ",d[c2][c]);
		}
		printf("\n");
	}
	return 0;
}

File: listas/python/Lista 4 sala/ex4.py
a = []
b = []
d = []
print("A: ")
for c in range(5):
    a.append(int(input(f"Digite o numero {c+1}: ")))

print("B: ")
for c in range(5):
    b.append(int(input(f"Digite o numero {c+1}: ")))

for c in range(10):
    if c<=4:
        d.append(a[c])
    else:
        d.append(b[c-5])

print("C: ")
for c in range(10):
    print(f'[{d[c]}]',end=" ")
